packages/expo-local-authentication/android/build.gradle

```diff
@@ -72,8 +72,8 @@ android {
 
   defaultConfig {
     minSdkVersion safeExtGet("minSdkVersion", 21)
-    targetSdkVersion safeExtGet("targetSdkVersion", 30)
-    versionCode 30
+    targetSdkVersion safeExtGet("targetSdkVersion", 31)
```

Comment:
Is changing `targetSdkVersion` necessary for this fix? Usually upgrading `compileSdkVersion` is enough. Upgrading `targetSdkVersion` may introduce some behavior changes for older SDK versions, so I'd like to double check if we really need it 😉